src/kakarot/evm.cairo

```diff
@@ -102,6 +102,19 @@ namespace EVM {
         );
     }
 
+    func out_of_gas(self: model.EVM*, revert_reason_len: felt, revert_reason: felt*) -> model.EVM* {
+        return new model.EVM(
```

Comment:
if it's out of gas, load the revert reason inside and don't take it as an argument